flake.nix

```diff
@@ -1,172 +1,122 @@
           inherit system;
-          overlays = [
-            (final: prev: {
-              esplora-electrs = prev.callPackage ./nix/esplora-electrs.nix {
-                inherit (prev.darwin.apple_sdk.frameworks) Security;
-              };
-
-              # syncing channels doesn't work right on newer versions, exactly like described here
-              # https://bitcoin.stackexchange.com/questions/84765/how-can-channel-policy-be-missing
-              # note that config-time `--enable-developer` turns into run-time `--developer` at some
-              # point
-              clightning = prev.clightning.overrideAttrs (oldAttrs: rec {
-                version = "23.05.2";
-                src = prev.fetchurl {
-                  url = "https://github.com/ElementsProject/lightning/releases/download/v${version}/clightning-v${version}.zip";
-                  sha256 = "sha256-Tj5ybVaxpk5wmOw85LkeU4pgM9NYl6SnmDG2gyXrTHw=";
-                };
-                makeFlags = [ "VERSION=v${version}" ];
-                configureFlags = [ "--enable-developer" "--disable-valgrind" ];
-                NIX_CFLAGS_COMPILE = "-w";
-              });
-            })
-          ];
+          overlays = fedimint.overlays.fedimint;
         };
-        fmLib = fedimint.lib.${system};
-        crane = fedimint.inputs.crane;
-        fenix = fedimint.inputs.fenix;
-        commonArgsBase = fmLib.commonArgsBase;
-
-
-        fenixChannel = fenix.packages.${system}.stable;
-
-        fenixToolchain = (fenixChannel.withComponents [
-          "rustc"
-          "cargo"
-          "clippy"
-          "rust-analysis"
-          "rust-src"
-          "llvm-tools-preview"
-        ]);
-
-        flakeboxLib = flakebox.lib.${system} {
-          # customizations will go here in the future
-          config = {
-            toolchain.components = [
-              "rustc"
-              "cargo"
-              "clippy"
-              "rust-analysis"
-              "rust-src"
-              "llvm-tools-preview"
-            ];
-
-            motd = {
-              enable = true;
-              command = ''
-                >&2 echo "🚧 In an enfort to improve documentation, we now require all structs and"
-                >&2 echo "🚧 and public methods to be documented with a docstring."
-                >&2 echo "🚧 See https://github.com/fedimint/fedimint/issues/3807"
-              '';
-            };
-            # we have our own weird CI workflows
-            github.ci.enable = false;
-            just.includePaths = [
-              "justfile.fedimint.just"
-            ];
-            # we have a custom final check
-            just.rules.final-check.enable = false;
-            git.pre-commit.trailing_newline = false;
-            git.pre-commit.hooks = {
-              check_forbidden_dependencies = builtins.readFile ./nix/check-forbidden-deps.sh;
-            };
+        lib = pkgs.lib;
+        flakeboxLib = flakebox.lib.${system} { };
```

Comment:
We should probably keep the previous `config` passed to `flakebox.lib.${system}`